src/docs/style/illustrations.mdx

```diff
@@ -0,0 +1,73 @@
+import DarkThemeImage from '../../app/assets/design-patterns/illustrations/illustration-dark-theme.svg';
+import BackgroundShapeImage from '../../app/assets/design-patterns/illustrations/illustration-background.png';
+
+<!-- KEYWORDS: design illustration graph onboarding -->
+
+# Illustrations
+
+Illustrations are design elements that should be used mostly for onboarding screens and [empty states](/patterns/empty-states). Do not use illustrations where you are supposed to use icons.
+
+When creating your own illustrations, you should follow our guidelines as explained below. Alternatively, you may also choose to acquire existing artworks from repositories like [Icons8](https://icons8.com/illustrations), [Adobe Stock](https://stock.adobe.com/vectors), or [Humaaans Figma plugin](https://www.figma.com/community/plugin/739503328703046360). We recommend you finalizing the illustration with [Adobe Illustrator](https://www.adobe.com/products/illustrator.html), as it provides more export options.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Onboarding'}
+        url={'https://material.io/design/communication/onboarding.html'}
+        description={`Material Design's guidelines on onboarding screens.`}
+    />
+    <MaterialDesignDescription
+        title={'Google Marketing Platform Visual Guidelines'}
+        url={
+            'https://designguidelines.withgoogle.com/ads-branding/assets/1hquKP2EcBlDaWyaLkoezqd8r9KPjRgnp/visual-guidelines-google-marketing-platform.pdf#'
+        }
+        description={`You are encouraged to follow their illustration guidelines.`}
+        avatar={<PDFIcon style={{ width: 40, height: 40 }} />}
+    />
+</React.Fragment>
+
+## Designing the Illustration
+
+### Perspective
+
+Your illustration should be either in a horizontal perspective (neither looking up nor looking down) or in parallel projection.
+
+### Ratio
+
+The width-height ratio should be between 1:1 to 2:3, which gives a margin on a mobile phone for UI interactions. If designing specifically for desktop applications (that is, the illustration will never show up on mobile), you can go wider than that, but no more than 16:9.
+
+### Drawing Style
+
+Keep your style consistent within one application, including your illustrations for the webpage for the product. Do not be too abstract or artistic, as your main goal is to inform your users of certain features in your product, instead of decorating it.
+
+### Text
+
+Avoid the use of text, as they might require translations, and can be hard to see on mobile. If you have to, use Open Sans, just like other PX Blue applications, and ensure that they are legit on the smallest targeted device.
+
+### Color
+
+Use [PX Blue palette](/style/color) as the base color. If there is a gradient, make sure the two ends of the gradient are using PX Blue colors. Use some blue hue (this can be colors from primary blue, Eaton blue, or light blue) somewhere in the foreground for your drawing.
```

Comment:
```suggestion
Use the [PX Blue color palette](/style/color) for your primary colors. You should use some shade of blue in the foreground of your images (these can be colors from the Blue, Light Blue, or Branding Blue palettes).

> When using gradients, both ends of the gradient should be set to colors from the PX Blue palette.
```

Reply:
When I say "base color", I was suggesting the color before people apply shadows to it (its own color)

Reply:
main colors? maybe kyle has another word for it?

Reply:
"local color"? https://en.wikipedia.org/wiki/Local_color_(visual_art)